REGRESSION (262629@main): Enabling UI-side compositing caused imageonly failure

karlrackler opened the pull request:
#### 54f075ab86d129c1006728f13870a4db3bdd414f
<pre>
REGRESSION (262629@main): Enabling UI-side compositing caused imageonly failure
<a href="https://bugs.webkit.org/show_bug.cgi?id=263083">https://bugs.webkit.org/show_bug.cgi?id=263083</a>
rdar://114328372

Unreviewed test modification.

Adjusting pixel tolerance.

* LayoutTests/fast/box-shadow/inset-spread-box-shadow-split-inline.html:

Canonical link: <a href="https://commits.webkit.org/269286@main">https://commits.webkit.org/269286@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/a832bba235bd0c154af64fde84b9d815aa6c70a6

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/22123 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/22331 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/23192 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/24015 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/20480 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/26582 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/22648 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/21515 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/22353 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/15/builds/22034 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/19193 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/24865 "Built successfully") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/39/builds/19115 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/10/builds/20049 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/26303 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/20134 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/20274 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/24162 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/7/builds/20706 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/17/builds/17624 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [❌ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/20060 "Failed to checkout and rebase branch from PR 19007") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/8/builds/19858 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/24268 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/2761 "Built successfully and passed tests") | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/31/builds/20654 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/9b6d247eb5b9f44b80470e389a4003d2d9c9fe6b. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/19007

webkit-commit-queue commented:
Commit message contains (OOPS!) and no reviewer found, blocking PR #19007

webkit-commit-queue commented:
Commit message contains (OOPS!) and no reviewer found, blocking PR #19007

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/a832bba235bd0c154af64fde84b9d815aa6c70a6. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/19007

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/a832bba235bd0c154af64fde84b9d815aa6c70a6. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/19007

karlrackler committed:
REGRESSION (262629@main): Enabling UI-side compositing caused imageonly failure
https://bugs.webkit.org/show_bug.cgi?id=263083
rdar://114328372

Unreviewed test modification.

Adjusting pixel tolerance.

* LayoutTests/fast/box-shadow/inset-spread-box-shadow-split-inline.html:

Canonical link: https://commits.webkit.org/269286@main

webkit-commit-queue commented:
Committed 269286@main (54f075ab86d1): <https://commits.webkit.org/269286@main>

Reviewed commits have been landed. Closing PR #19007 and removing active labels.
